### PR TITLE
Feature/elasticsearch storage

### PR DESCRIPTION
**Pull request by mob8607**

### CODE REVIEW

File: src/Repository/UserRepositoryInterface.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
```
Comment (wachterjohannes) [inline src/Repository/UserRepositoryInterface.php]: header

File: src/Repository/UserRepositoryInterface.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+namespace Sulu\Component\ActivityLog\Repository;
+
+use Symfony\Component\Security\Core\User\UserInterface;
+
+interface UserRepositoryInterface
```
Comment (wachterjohannes) [inline src/Repository/UserRepositoryInterface.php]: phpdoc

**Review by wachterjohannes — CHANGES_REQUESTED** [review]
else ACK

**Comment by mob8607** [comment]
@wachterjohannes updated

**Pull request merged**